backend/CircleForms/Controllers/PostsController.cs

```diff
@@ -40,13 +30,47 @@ public async Task<IActionResult> Post(Post post)
         if (!string.IsNullOrEmpty(claim) && long.TryParse(claim, out var userId))
         {
             post.AuthorId = userId;
-            var result = await _postRepository.AddPost(userId, post);
+            var result = await _postRepository.Add(userId, post);
 
-            return Ok(result);
+            return CreatedAtAction("Get", new {id = post.Id.ToString()}, result);
         }
 
         _logger.LogWarning("User had an invalid name claim: {Claim}", claim);
 
         return BadRequest();
     }
+
+    #region Mongo
+    [HttpGet("/posts/mongo/{id}")]
+    public async Task<Post> Get(string id)
```

Comment:
Do we even need to expose that? And why not use a query param on a regular get, for example `/posts/1?cached=false`?

Reply:
Keep in mind it's gonna be public, so anyone would be able to use uncached endpoint and potentially overload the database

Reply:
We are not exposing it, see #35